service/src/test/java/bio/terra/workspace/app/controller/WorkspaceApiControllerTest.java

```diff
@@ -369,11 +374,7 @@ public void cloneWorkspace_rawls() throws Exception {
 
     ApiCloneWorkspaceResult cloneWorkspace =
         mockWorkspaceV1Api.cloneWorkspace(
-            USER_REQUEST,
-            sourceWorkspaceId,
-            DEFAULT_SPEND_PROFILE_NAME,
-            null,
-            destinationWorkspaceId);
+            USER_REQUEST, sourceWorkspaceId, null, null, destinationWorkspaceId);
```

Comment:
This tests the case of not having a spend profile on clone. Do we have coverage elsewhere for having a profile (and linking the policy attributes)?

Reply:
The test above covers cloning with a profile. I also added a verify to this test to ensure that we're merging the policy from source -> destination correctly.